src/test/java/seedu/address/logic/parser/RescheduleMeetingCommandParserTest.java

```diff
@@ -0,0 +1,93 @@
+        assertParseSuccess(parser, FIRST_PERSON_INDEX.getOneBased() + " "
+                        + PREFIX_MEETING_INDEX + FIRST_MEETING_INDEX.getOneBased() + " "
+                + PREFIX_MEETING_DATE + VALID_DATE + " "
+                + PREFIX_MEETING_TIME + VALID_TIME + " ",
+                new RescheduleMeetingCommand(FIRST_PERSON_INDEX, FIRST_MEETING_INDEX, meetingDateTime));
+
+    }
+    @Test
+    public void parse_missingCompulsoryFields_failure() {
+        // Missing time input
+        assertParseFailure(parser,FIRST_PERSON_INDEX.getOneBased() + " "
+                        + PREFIX_MEETING_INDEX + FIRST_MEETING_INDEX.getOneBased() + " "
+                        + PREFIX_MEETING_DATE + VALID_DATE + " ",
+                 "Meeting index, date and time are required for scheduling a meeting.");
+        // Missing date input
+        assertParseFailure(parser,FIRST_PERSON_INDEX.getOneBased() + " "
+                        + PREFIX_MEETING_INDEX + FIRST_MEETING_INDEX.getOneBased() + " "
+                        + PREFIX_MEETING_TIME + VALID_TIME + " ",
+                 "Meeting index, date and time are required for scheduling a meeting.");
+        // Missing meeting index input
+        assertParseFailure(parser,FIRST_PERSON_INDEX.getOneBased() + " "
+                        + PREFIX_MEETING_DATE + VALID_DATE + " "
+                        + PREFIX_MEETING_TIME + VALID_TIME + " ",
+                 "Meeting index, date and time are required for scheduling a meeting.");
+    }
+    @Test
+    public void parse_missingPersonIndex_failure() {
+        assertParseFailure(parser," " +
+                        PREFIX_MEETING_INDEX + FIRST_MEETING_INDEX.getOneBased() + " "
+                + PREFIX_MEETING_DATE + VALID_DATE + " "
+                + PREFIX_MEETING_TIME + VALID_TIME + " " ,
+                String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                RescheduleMeetingCommand.MESSAGE_USAGE));
+
+    }
+    @Test
+    public void parse_invalidDateTime_failure() {
+        // Invalid dates
+
+        assertParseFailure(parser, FIRST_PERSON_INDEX.getOneBased() + " "
+                + PREFIX_MEETING_INDEX + FIRST_MEETING_INDEX.getOneBased() + " "
+                + PREFIX_MEETING_DATE + INVALID_DATE + " "
+                + PREFIX_MEETING_TIME + VALID_TIME + " ",
+                String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                        RescheduleMeetingCommand.MESSAGE_USAGE));
+        // Invalid Time
+        assertParseFailure(parser, FIRST_PERSON_INDEX.getOneBased() + " "
+                        + PREFIX_MEETING_INDEX + FIRST_MEETING_INDEX.getOneBased() + " "
+                        + PREFIX_MEETING_DATE + VALID_DATE + " "
+                        + PREFIX_MEETING_TIME + INVALID_TIME+ " ",
+                String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                        RescheduleMeetingCommand.MESSAGE_USAGE));
+    }
+}
```

Comment:
The test parse_invalidDateTime_failure effectively covers scenarios with invalid dates and times. However, it might be more informative to split this into two separate tests: one for invalid dates and one for invalid times. This way, it's clearer which specific part of the date-time parsing is failing.